src/foam/nanos/auth/Country.js

```diff
@@ -21,5 +21,14 @@ foam.CLASS({
       class: 'String',
       name: 'name'
     }
-  ]
+  ],
+  methods: [{
+    name: 'toString',
+    returns: 'String',
+    javaReturns: 'String',
```

Comment:
Does this mess up the JS toString()?

Reply:
I don’t think it will mess up js toString, since this is javaCode, right?